spec/forms/register_user_email_form_spec.rb

```diff
@@ -33,6 +34,12 @@
       end
 
       it 'creates throttle events after reaching throttle limit' do
+        expect(attempts_tracker).to receive(
+          :user_registration_email_submission_rate_limited,
+        ).with(
+          { email: 'taken@example.com', email_confirmed_with_password: true },
+        )
```

Comment:
I think we can drop the curly braces? (and the in rest of the file too)
```suggestion
        ).with(email: 'taken@example.com', email_confirmed_with_password: true)
```